### PR TITLE
Remove CC for Video Background Effects

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Remove CC for Video Background Effects

# Why
[BUG 3627516](https://skype.visualstudio.com/SPOOL/_workitems/edit/3627516)

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/calling-stateful-client/src/Converter.ts [hunk 1]
```diff
@@ -27,11 +27,9 @@ import { CaptionsInfo } from './CallClientState';
 import { _isACSCall } from './TypeGuards';
 import { CallCommon, IncomingCallCommon } from './BetaToStableTypes';
 
-/* @conditional-compile-remove(video-background-effects) */ /* @conditional-compile-remove(optimal-video-count) */
+/* @conditional-compile-remove(optimal-video-count) */
```
Comment (dmceachernmsft) [inline packages/calling-stateful-client/src/Converter.ts:30]: We should be removing this CC line as well (also stabilized but will make this cleaner)

File: packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts [hunk 2]
```diff
@@ -46,15 +46,14 @@ import { TeamsCaptions } from '@azure/communication-calling';
...
-/* @conditional-compile-remove(video-background-effects) */
 import type { BackgroundBlurConfig, BackgroundReplacementConfig } from '@azure/communication-calling';
 /* @conditional-compile-remove(capabilities) */
 import type { CapabilitiesChangeInfo } from '@azure/communication-calling';
 /* @conditional-compile-remove(teams-identity-support)) */
 import { TeamsCallAgent } from '@azure/communication-calling';
 /* @conditional-compile-remove(rooms) */
 import { RoomCallLocator } from '@azure/communication-calling';
-/* @conditional-compile-remove(unsupported-browser) */ /* @conditional-compile-remove(video-background-effects) */ /* @conditional-compile-remove(close-captions) */ /* @conditional-compile-remove(capabilities) */
+/* @conditional-compile-remove(unsupported-browser) */ /* @conditional-compile-remove(close-captions) */ /* @conditional-compile-remove(capabilities) */
```
Comment (dmceachernmsft) [inline packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts:56]: I think all these can be removed as well

File: packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx [hunk 3]
```diff
@@ -102,9 +99,8 @@ export interface LocalDeviceSettingsType {
 export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element => {
   const theme = useTheme();
   const locale = useLocale();
-  /* @conditional-compile-remove(call-readiness) */ /* @conditional-compile-remove(video-background-effects) */ /* @conditional-compile-remove(rooms) */
+  /* @conditional-compile-remove(call-readiness) */ /* @conditional-compile-remove(rooms) */
```
Comment (dmceachernmsft) [inline packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx:102]: Same here you should be able to remove these as well

File: samples/Calling/src/app/views/CallScreen.tsx [hunk 4]
```diff
@@ -176,7 +173,7 @@ const AzureCommunicationCallScreen = (props: AzureCommunicationCallScreenProps):
       ...adapterArgs,
       userId,
       locator,
-      /* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(unsupported-browser) */ /* @conditional-compile-remove(video-background-effects) */
+      /* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(unsupported-browser) */
```
Comment (dmceachernmsft) [inline samples/Calling/src/app/views/CallScreen.tsx:176]: Same here as previous comments

File: samples/Calling/src/app/views/CallScreen.tsx [hunk 5]
```diff
@@ -222,7 +218,7 @@ const AzureCommunicationOutboundCallScreen = (props: AzureCommunicationCallScree
       ...adapterArgs,
       userId,
       targetCallees: targetCallees,
-      /* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(unsupported-browser) */ /* @conditional-compile-remove(video-background-effects) */
+      /* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(unsupported-browser) */
```
Comment (dmceachernmsft) [inline samples/Calling/src/app/views/CallScreen.tsx:221]: And here 😊

**Review by dmceachernmsft — APPROVED** [review]
Couple CC comments that could also be removed for code cleanliness but not blocking as they will be removed by other features as well

File: packages/calling-component-bindings/src/handlers/createCommonHandlers.ts [hunk 6]
```diff
@@ -32,7 +32,7 @@ import { disposeAllLocalPreviewViews, _isInCall, _isInLobbyOrConnecting, _isPrev
 /* @conditional-compile-remove(PSTN-calls) */
 import { CommunicationUserIdentifier, PhoneNumberIdentifier } from '@azure/communication-common';
 import { CommunicationIdentifier } from '@azure/communication-common';
-/* @conditional-compile-remove(video-background-effects) */ /* @conditional-compile-remove(close-captions) */ /* @conditional-compile-remove(raise-hand) */ /* @conditional-compile-remove(end-of-call-survey) */
+/* @conditional-compile-remove(close-captions) */ /* @conditional-compile-remove(raise-hand) */ /* @conditional-compile-remove(end-of-call-survey) */
```
Comment (edwardlee-msft) [inline packages/calling-component-bindings/src/handlers/createCommonHandlers.ts:35]: This one as well requires you to remove all the other cc comments.

File: packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx [hunk 7]
```diff
@@ -18,14 +17,13 @@ import {
   optionIconStyles,
   soundStackTokens
 } from '../styles/LocalDeviceSettings.styles';
-/* @conditional-compile-remove(call-readiness) */ /* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(video-background-effects) */
+/* @conditional-compile-remove(call-readiness) */ /* @conditional-compile-remove(rooms) */
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx:20]: Same here need to remove the other cc comments

File: samples/Calling/src/app/views/CallScreen.tsx [hunk 8]
```diff
@@ -15,9 +15,8 @@ import {
 } from '@azure/communication-react';
 /* @conditional-compile-remove(teams-identity-support) */
 import { useTeamsCallAdapter, TeamsCallAdapter } from '@azure/communication-react';
-/* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(video-background-effects) */
+/* @conditional-compile-remove(rooms) */
```
Comment (edwardlee-msft) [inline samples/Calling/src/app/views/CallScreen.tsx:18]: Same here need to remove the other cc comments

**Pull request closed without merging**